config.yaml.defaults

```diff
@@ -156,3 +167,5 @@ security:
 #   - interval: 5
 #     script: tools/5_minute.py
 #     limit: ["01:00", "02:00"]
+
+plugins: {}
```

Comment:
Why plugins, with external services already defined?

Reply:
Deleted, it was a mistake.